RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: minor
+
+This release deprecates Hypothesis's strict mode, which turned Hypothesis's
+deprecation warnings into errors. Similar functionality can be achieved by
+using the standard library's warnings.simplefilter directly.
```

Comment:
by using :func:`simplefilter('error', HypothesisDeprecationWarning) <python:warnings.simplefilter>` directly.

(I like explicit advice on how to update, and as many cross-references as possible)